user/stubs.c

```diff
@@ -0,0 +1,119 @@
+#include "userargs.h"
+#include <sys/syscall.h>
+
+sgxlkl_userargs_t* __sgxlkl_userargs;
+
+void sgxlkl_warn(const char* fmt, ...);
+
+int snprintf(char *str, size_t size, const char *format, ...);
+
+/*
+**==============================================================================
+**
+** syscall:
+**
+**==============================================================================
+*/
+
+long lkl_syscall(long no, long* params)
+{
+    long ret = __sgxlkl_userargs->ua_lkl_syscall(no, params);
+
+    return ret;
+}
```

Comment:
Fine for now, but please open an issue to track the fact that we should be always referencing `lkl_syscall` via a function pointer (or, ideally, an ifunc).